Sources/Foundation/UserDefaults.swift

```diff
@@ -60,19 +58,15 @@ open class UserDefaults: NSObject {
     }
     
     static private func _unboxingNSNumbers(_ value: Any?) -> Any? {
-        if value == nil {
+        guard value != nil else {
             return nil
         }
         
         if let number = value as? NSNumber {
             return number._swiftValueOfOptimalType
-        }
-        
-        if let value = value as? [Any] {
+        } else if let value = value as? [Any] {
             return value.map(_unboxingNSNumbers)
-        }
-        
-        if let value = value as? [AnyHashable: Any] {
+        } else if let value = value as? [AnyHashable: Any] {
             return value.mapValues(_unboxingNSNumbers)
```

Comment:
As long as we `return` in every `if` body, `else if` would not make codes better to read. IMO this would just introduce unnecessary inconsistency between these parallel `if`s.

If you want to make the codes shorter and more verbose, use `switch` instead.